build.js

```diff
@@ -50,364 +50,641 @@ var androidVersion = require('./package.json').version;
+    newText += text.substring(end);
+    if(replaceText.includes('google-services')) {
+    }
+    return newText;
 };
 
-function injectAppLinks(outputPath, android_manifest) {
-  var manifestXml = path.join(outputPath, 'AndroidManifest.xml');
-  var app_links = android_manifest.app_links;
-  var template = '<data android:host="curr_host" android:scheme="curr_scheme"/>';
-  var result = '';
-
-  if (!app_links || app_links.length === 0) {
-    return Promise.resolve(true);
-  }
-
-  app_links.forEach(function (curr) {
-    var curr_data = '';
-    var host = curr.host;
-    var scheme = curr.scheme || 'http';
-
-    curr_data = template.replace('curr_host', host);
-    curr_data = curr_data.replace('curr_scheme', scheme);
-    ['path', 'pathPrefix', 'pathPattern', 'port'].forEach(function (prop) {
-      if (curr[prop]) {
-        curr_data = curr_data.replace('/>', ' android:'.concat(prop, '="', curr[prop], '"/>'))
-      }
-    });
-    result = result.concat(curr_data);
-  });
-  result = result.concat('\n');
+function injectAppLinks(android_manifest, app) {
+    var manifestXml = path.join(projectPath ,"/app/src/main",  'AndroidManifest.xml');
+    var app_links = android_manifest.app_links;
+    var template = '<data android:host="curr_host" android:scheme="curr_scheme"/>';
+    var result = '';
 
-  return Promise.resolve()
-    .then(function () {
-      return fs.readFileAsync(manifestXml, 'utf-8');
-    })
-    .then(function (xml) {
-      var XML_START_PLUGINS_LINKS = '<!--START_PLUGINS_LINKS-->';
-      var XML_END_PLUGINS_LINKS = '<!--END_PLUGINS_LINKS-->';
+    if (!app_links || app_links.length === 0) {
+        return Promise.resolve(true);
+    }
 
-      xml = replaceTextBetween(xml, XML_START_PLUGINS_LINKS, XML_END_PLUGINS_LINKS, result);
+    app_links.forEach(function (curr) {
+        var curr_data = '';
+        var host = curr.host;
+        var scheme = curr.scheme || 'http';
 
-      return fs.writeFileAsync(manifestXml, xml, 'utf-8');
+        curr_data = template.replace('curr_host', host);
+        curr_data = curr_data.replace('curr_scheme', scheme);
+        ['path', 'pathPrefix', 'pathPattern', 'port'].forEach(function (prop) {
+            if (curr[prop]) {
+                curr_data = curr_data.replace('/>', ' android:'.concat(prop, '="', curr[prop], '"/>'))
+            }
+        });
+        result = result.concat(curr_data);
     });
+    result = result.concat('\n');
+
+    return Promise.resolve()
+        .then(function () {
+            return fs.readFileAsync(manifestXml, 'utf-8');
+        })
+        .then(function (xml) {
+            var XML_START_PLUGINS_LINKS = '<!--START_PLUGINS_LINKS-->';
+            var XML_END_PLUGINS_LINKS = '<!--END_PLUGINS_LINKS-->';
+
+            xml = replaceTextBetween(xml, XML_START_PLUGINS_LINKS, XML_END_PLUGINS_LINKS, result);
+
+            return fs.writeFileAsync(manifestXml, xml, 'utf-8');
+        });
 };
 
-function injectPluginXML(opts) {
```

Comment:
looks like all changes above this line for this file is just indentation changes. am i right ?

It will be more convenient to review if the indentation changes can be removed.

Reply:
@bijoshtj  use ?w=1 at the end of the url to ignore whitespace changes. Would be easier to review that way